javatests/google/registry/rde/imports/RdeHostLinkActionTest.java

```diff
@@ -87,9 +89,17 @@ public void test_mapreduceSuccessfullyLinksHost() throws Exception {
     Key<DomainResource> superOrdinateDomainKey = Key.create(superordinateDomain);
     pushToGcs(DEPOSIT_1_HOST);
     runMapreduce();
+    // verify that host is linked to domain
     List<HostResource> hosts = ofy().load().type(HostResource.class).list();
     assertThat(hosts).hasSize(1);
     assertThat(hosts.get(0).getSuperordinateDomain()).isEqualTo(superOrdinateDomainKey);
+    // verify that last superordinate change is within the last 10 seconds
+    assertThat(hosts.get(0).getLastSuperordinateChange())
+        .isAtLeast(DateTime.now().minus(Seconds.seconds(10)));
```

Comment:
ofy.transactionTime() is not the same as what is returned from the injected clock...